.travis.yml

```diff
@@ -4,6 +4,7 @@ before_script: ./script/ci/before_build.sh
 rvm:
   - 2.6.1
   - 2.6.5
+  - 2.7.1
```

Comment:
let's be crazy and use the latest one (2.7.2)

Reply:
with pleasure

Reply:
3.0.0.pre1 is out though but even I am not that mad